models/semester.js

```diff
@@ -0,0 +1,51 @@
+import connector from "#models/databaseUtil";
+
+const semesterSchema = {
+  number:{intenum: ["1", "2" , "3" , "4" , "5","6","7","8"],required:true},
```

Comment:
check the file for ESLint, and fix this line especially, intenum isnt a thing afaik